### PR TITLE
RSO: Import data symbols

### PR DESCRIPTION
**Pull request by sepalani**
This PR allows to import data symbols from RSO modules.

Ready to be reviewed & merged.

### CODE REVIEW

**Comment by sepalani** [comment]
@dolphin-emu-bot rebuild

File: Source/Core/Core/Debugger/RSO.cpp [hunk 1]
```diff
@@ -380,14 +379,21 @@ void RSOView::Apply(PPCSymbolDB* symbol_db) const
...
       {
-        symbol->name = GetExportName(rso_export);
-        count += 1;
+        // Function symbol
+        symbol->name = export_name;
+      }
+      else
+      {
+        // Data symbol
+        symbol_db->AddKnownSymbol(address, 0, export_name, Symbol::Type::Data);
```
Comment (leoetlino) [inline Source/Core/Core/Debugger/RSO.cpp]: Not very familiar with RSO, but is there a reason the size is always 0?

Comment (sepalani) [inline Source/Core/Core/Debugger/RSO.cpp]: The main reason is because there is no symbol size field, AFAIK. The goal of this PR is to show these symbols in the memory view (just like ELF reader does by adding data symbol).

In theory, there is a hash field for each (export) symbol that supposedly correspond to the hash of said symbol data. If that's true and I know how the hash is computed, I could try to use it in order to determine the size of each symbol. However, that's a bit overkill for the amount of symbol and just for displaying them on the memory view.

Comment (leoetlino) [inline Source/Core/Core/Debugger/RSO.cpp]: Ah okay. Might be best to leave that for another PR then, when we figure out how the hash is generated.
